prediction_prophet/autonolas/research.py

```diff
@@ -1182,6 +1182,8 @@ def make_prediction(
     additional_information: str,
     temperature: float = 0.7,
     engine: str = "gpt-3.5-turbo-0125",
+    log_probs: bool = False,
```

Comment:
Getting probability from log_probs was an experiment and isn't used in the end. Plus it's not supported by new models, so let's just default to False.